Core/src/org/sleuthkit/autopsy/discovery/ui/Bundle.properties-MERGED

```diff
@@ -66,12 +68,18 @@ ImageThumbnailPanel.isDeleted.text=All instances of file are deleted.
 # {0} - otherInstanceCount
 ImageThumbnailPanel.nameLabel.more.text=\ and {0} more
 InterestingItemsFilterPanel.error.text=At least one interesting file set name must be selected.
+LoadingPanel.loading.text=Loading, please wait.
+# {0} - resultInfo
+LoadingPanel.retrieving.text=Retrieving results for {0}.
 MiniTimelineArtifactListPanel.descriptionColumn.name=\ Description
 MiniTimelineArtifactListPanel.typeColumn.name=Result Type
 MiniTimelineArtifactListPanel.value.noValue=No value available.
 MiniTimelineDateListPanel.countColumn.name=Count
 MiniTimelineDateListPanel.dateColumn.name=Date
 MiniTimelineDateListPanel.value.noValue=No value available.
+<<<<<<< HEAD
```

Comment:
I think this might need to be fixed along with line 82 and 95.

Reply:
Yes, good catch. I needed a clean and build to get the correct version of the file, and it also reminded me that we wanted to change the display name to just Timeline, I have left the classes and developer facing items named with mini to minimize the effect of further overloading of terms. Thank you.